Test.C

```diff
@@ -130,40 +122,45 @@ int main(){
     // Setup player
     player.equipmentInventory[0] = 1;                  /* TODO: RANDOMLY POPULATE EQUIPMENT (Do we need to tier equipment?) */
     player.encryptionInventory = {encryptionStock[0]}; /* TODO: RANDOMLY POPULATE ENCRYPTION (Do we need to tier encryption?) */
-    player.currentCredits = 30;
+    player.currentCredits = 1000;
 
     // output pregame information
     displayBriefingMessage(&player);
 
     while (currentRound < NUM_ROUNDS){
+        printf("\n");
         printf("|================================== Round %2d ==================================|\n", currentRound);
```

Comment:
```suggestion
        printf("|================================== Round %2d ==================================|\n", currentRound + 1);
```

Reply:
Makes round numbering display in a way that makes sense without adjusting logic of loops